lib/fs.js

```diff
@@ -145,8 +145,8 @@ function makeCallback(cb) {
     throw new errors.TypeError('ERR_INVALID_CALLBACK');
   }
 
-  return function() {
-    return cb.apply(undefined, arguments);
+  return function(...arguments) {
```

Comment:
This should be `...args` to match the variable below.

Reply:
@apapirovski : my bad.. will fix that !